frame/staking/src/pallet/mod.rs

```diff
@@ -90,6 +90,7 @@ pub mod pallet {
 			+ sp_std::fmt::Debug
 			+ Default
 			+ From<u64>
+			+ Into<u64>
```

Comment:
why is this needed?

Reply:
Just for convenience to pass the deposit from UnstakeRequest `deposit.into()` to balance operations.

Reply:
I'm sure there must be a better solution to this

Reply:
The initial reason was the fact that passing `T: pallet_staking::Config` to `UnstakeRequest` results in a `TypeInfo` generation error.